### ISSUE TITLE
which jellyfish version

### ISSUE THREAD

**Issue by mictadlo** [DerrickWood/kraken#61]
Hi,
Which Jellyfish version is compatible to kraken:

```
> conda search jellyfish
Fetching package metadata ...............
jellyfish                    0.5.1                    py27_0  conda-forge     
                             0.5.1                    py34_0  conda-forge     
                             0.5.1                    py35_0  conda-forge     
                             0.5.6                    py35_0  conda-forge     
                             0.5.6                    py27_0  conda-forge     
                             0.5.6                    py34_0  conda-forge     
                             1.1.11                        0  bioconda        
                             1.1.11                        1  bioconda        
                             2.2.3                         0  bioconda        
                             2.2.3                         1  bioconda        
                          *  2.2.6                         0  bioconda  
```
Thank you in advance.

Michal

**Comment by ressy** [comment]
Hi Michal,

I'm setting up an environment to run Kraken also, and trying to figure this out.  When I installed the latest Kraken release (0.10.5-beta) I saw that the [manual](http://ccb.jhu.edu/software/kraken/MANUAL.html#system-requirements) says it requires Jellyfish version 1 and explicitly not version 2.  I installed 1.1.11 since that's the latest of version 1 that exists.  For me, Kraken stopped complaining to me about Jellyfish so I assumed it was happy with it, though I'm seeing some errors when building a custom database.  (Not sure if that's related though.)

So, I'll be curious to know if there are any more specific requirements for the Jellyfish version, but version 1 is what the documentation says for the latest kraken release.

Jesse
